command_line.go

```diff
@@ -26,6 +26,7 @@ func ParseCommandLine(app *kingpin.Application) (*cmd.CommandLine, error) {
 	// 2. Parse flags
 	cmdLine := cmd.CommandLine{}
 	app.Flag("config", "path to rsc config file").Short('c').Default(path.Join(os.Getenv("HOME"), ".rsc")).StringVar(&cmdLine.ConfigPath)
+	app.Flag("retry", "number that rsc would retry on non-successful API response").Short('R').Default("1").IntVar(&cmdLine.Retry)
```

Comment:
Can we have this default to zero? I think the client should always try once and if i pass the command retry 1 it should do the first attempt and then if that fails it should retry that call, which results in a total of 2 tries.